execution_engine/src/storage/global_state/lmdb.rs

```diff
@@ -71,6 +74,32 @@ impl LmdbGlobalState {
             empty_root_hash,
         }
     }
+
+    /// Creates an in-memory cache for changes written.
+    pub fn create_scratch(&self) -> ScratchGlobalState {
+        ScratchGlobalState::new(
+            Arc::clone(&self.environment),
+            Arc::clone(&self.trie_store),
+            self.empty_root_hash,
```

Comment:
So I'm going to recommend that we make this a constant rather than carry it around.

It's the blake2b digest of the empty pointer block.

Reply:
Well this field may be named inappropriately. An empty state root hash (a single pointer block `Trie::Node` with no entries) which is hashed is written to LMDB, and then that hash passed along inside LmdbGlobalState. This makes this task more than just lifting a constant out - we would need to ensure that the state root exists in the database. That said I understand that the hash would always be the same - however I'd argue that this current path leads developers back to the source of truth, without having a magic constant and an assumed-to-be-present database record.

Reply:
It’s a confusing wart, so I guess there’s no nice way about this.